roles/openshift_storage_nfs_lvm/tasks/nfs.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: Install NFS server
   yum: name=nfs-utils state=present
+  when: not is_containerized | bool
```

Comment:
I think we can just fail on this role when is_containerized is true. It isn't actively used anywhere in our playbooks and I suspect it'll have more issues than just the nfs package being installed to work containerized.